tests/integration/targets/nxos_vrf_interfaces/tests/common/gathered.yaml

```diff
@@ -16,7 +16,9 @@
       ansible.builtin.assert:
         that:
           - not result.changed
-          - "{{ gathered['config'] | symmetric_difference(result['gathered']) |length == 0 }}"
+          # Since different devices may have different interfaces, we can't assert the exact number of interfaces
```

Comment:
These tests are _only_ supposed to work on a very specific chassis x NX-OS combination - both upstream and downstream. In that case, this shouldn't be an issue.

Reply:
currently when testing, i was having upto 64 interfaces in the downstream device, we would have to add all these 64 interfaces in our vars/main.yaml for every list we create hence im asserting if the change i want exist instead of adding all of them

Reply:
Does the upstream one have 64 interfaces as well? If so, then we can just check for N interfaces that are touched by this role.

Reply:
yes the upstream also has 64 interfaces, the above change only checks for those n interfaces that have a change in them